C/eval.c

```diff
@@ -306,17 +306,42 @@ typedef struct evalState {
   frameItem* activeWriteFrame;
 } evalState;
 
+typedef unsigned char flags_type;
 /* 'call' is an item is used to track the "call stack" of the Bit Machine during evaluation.
- * Each call stack frame holds the call's TCO state and where to return to after the call.
+ * Each call stack frame remembers where to return to after the call and a set of flags to hold various bits of state.
  */
 typedef struct call {
   size_t return_to;
-  union {
-    bool tcoOn;
-    bool bit; /* Scratch space used by the 'CASE' combinator. */
-  };
+  flags_type flags;
 } call;
 
+#define TCO_FLAG        0x01 // Whether TCO is on (1) or off (0).
+#define CASE_LAST_FLAG  0x02 // For case combinators, what was the last branch executed was right (1) or left (0).
```

Comment:
In ed837d59f01fe25cd327d55b5753d09051e23f4c:

nit: `what was the` should be removed

Reply:
Done.